clients/fides-js/src/components/fides.css

```diff
@@ -97,7 +98,7 @@ div#fides-banner-container.fides-banner-bottom {
 }
 
 div#fides-banner-container.fides-banner-bottom.fides-banner-hidden {
-  transform: translateY(100%);
+  transform: translateY(150%);
```

Comment:
ideally this would be `100% + 48px` but I couldn't find a way to combine them. 150% seems to work well in this case though